### PR TITLE
[DOCFIX] Add docs Deploy-Alluxio-Fuse-for-Deep-Learning.md

### PR DESCRIPTION
**Pull request by iluoeli**

### CODE REVIEW

File: docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+        <td>kernel_cache</td>
+        <td></td>
+        <td>建议启用</td>
+        <td>开启page_cache，使用更多的系统缓存，同时提升文件读取速度。</td>
+    </tr>
+    <tr>
+        <td>max_read=N</td>
+        <td>131072</td>
+        <td>131072</td>
+        <td>FUSE单次request能读取文件的大小上限。在kernel 4.19中，这个值是32个pages，在i386上，为131072，或者说128kbytes。</td>
```
Comment (cheyang) [inline docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md:177]: 为什么要强调在 kernel 4.19，这个值是32pages？还是说在kernel 5.0之前都是32 pages，128kbytes？

Comment (iluoeli) [reply docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md:177]: 因为实验环境是4.19的kernel，所以就写上去了，确实没必要在这里刻意说4.19

File: docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+nickname: 用AlluxioFuse加速深度学习训练（Experimental）
+group: Install Alluxio
+priority: 5
+---
+
+* 内容列表
+{:toc}
+## 1. Overview
+
+### 什么是Alluxio
```
Comment (apc999) [inline docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md:13]: don't need this section of `什么是Alluxio`

File: docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md [hunk 3]
```diff
@@ -0,0 +1,492 @@
...
+layout: global
+title: 用AlluxioFuse加速深度学习训练（Experimental）
+nickname: 用AlluxioFuse加速深度学习训练（Experimental）
+group: Install Alluxio
+priority: 5
+---
+
+* 内容列表
+{:toc}
+## 1. Overview
```
Comment (apc999) [inline docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md:11]: Remove the order "1." and in following section titles. They are auto generated.

File: docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md [hunk 4]
```diff
@@ -0,0 +1,492 @@
...
+注意：我们正在持续解决和优化AlluxioFuse的稳定性和速度等问题，因此，现在提供的并不是正式版，而是实验版本，欢迎试用并提出建议。
+
+## 2. 在Docker上单机部署
+
+### 2.1 前提条件
+
+- [Docker](https://www.docker.com/)
+- Aluxio Docker镜像
+
+其中，Alluxio Docker镜像直接使用阿里云镜像，镜像信息如下表所示：
```
Comment (apc999) [inline docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md:32]: mention why we need two different images?

File: docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md [hunk 5]
```diff
@@ -0,0 +1,492 @@
...
+$ mkdir underStorage
+```
+
+#### 2.2.2 设置RAMFS
+
+在主机上创建文件夹`/mnt/ramdisk`，并挂载为`ramfs`，注意添加ramfs的读写权限
+
+``` console
+$ sudo mkdir /mnt/ramdisk
+$ sudo mount -t ramfs -o size=1G ramfs /mnt/ramdisk
```
Comment (apc999) [inline docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md:66]: mention then ramfs is 1GB in this example?

File: docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md [hunk 6]
```diff
@@ -0,0 +1,492 @@
...
+
+```console
+$ mkdir underStorage
+```
+
+#### 2.2.2 设置RAMFS
+
+在主机上创建文件夹`/mnt/ramdisk`，并挂载为`ramfs`，注意添加ramfs的读写权限
+
+``` console
```
Comment (apc999) [inline docs/cn/deploy/Deploy-Alluxio-Fuse-for-Deep-Learning.md:64]: remove space before `console`

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**